packages/design-system/src/components/ButtonIcon/ButtonIcon.stories.mdx

```diff
@@ -0,0 +1,198 @@
+import { ArgsTable, Meta, Canvas, Story } from '@storybook/addon-docs';
+import { FigmaImage, FigmaLink, Links } from '../../docs';
+import ButtonIconPrimitive from './Primitive/ButtonIconPrimitive';
+import * as Stories from './ButtonIcon.stories';
+
+<Meta title="Components/ButtonIcon" component={ButtonIconPrimitive} />
+
+<Links>
+	<FigmaLink href="https://www.figma.com/file/KuJ1PlP77uyXlfOhdniqsZ/ButtonIcon?node-id=1%3A75" />
+</Links>
+
+# ButtonIcon
+
+This component should be used when icons are meant to be clicked on.
+
+It handles the two largest usecases we have for clickable icons: actions and toggles.
+
+- Always try use when an icon is meant to be clicked on.
+- Always provide developers with copy for the tooltip.
+- If the button holds the ON / OFF state of something, then use the Toggle variants.
+- If the button is floating on top of something else, use the Floating variant.
+- Only use size XS sporadically.
+- Spacing between buttons should be at least 4px (XXS), 8px is best (XS).
+
+## Zoning
+
+<FigmaImage
+	src="https://www.figma.com/file/qc8oUSyVJM67gSsbTShftt/%F0%9F%93%90-Zoning?node-id=1049%3A415"
+	alt="zoning image for ButtonIcons"
+/>
+
+## Style
+
+### Variations
+
+<FigmaImage
+	src="https://www.figma.com/file/KuJ1PlP77uyXlfOhdniqsZ/ButtonIcon?node-id=3232%3A34023"
```

Comment:
No access to the link here neither. But maybe it is normal and FigmaImage is able to resolve it?